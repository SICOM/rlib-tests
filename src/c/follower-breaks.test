COMMAND=./follower-breaks
FORMATS="txt"
EXPECTED_pdf=
EXPECTED_xml=
EXPECTED_txt=
EXPECTED_csv=
EXPECTED_html=
