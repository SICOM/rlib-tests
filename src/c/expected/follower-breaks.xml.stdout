<?xml version="1.0" encoding="UTF-8" ?>
<rlib>
<part>
<part_page_footer>
<output>
<line>
<data col="0" width="6" font_point="9" bold="0" italics="0" align="left" >Page: </data>
<data col="0" width="3" font_point="9" bold="0" italics="0" align="right" type="number" >  0</data>
</line>
</output>
</part_page_footer>
<part_header>
<output>
<line>
<data col="0" width="47" font_point="11" bold="0" italics="0" align="left" >Test Report (Normal) AND THE REPORT FORMAT IS: </data>
<data col="0" width="10" font_point="11" bold="0" italics="0" align="left" type="string" >XML       </data>
<data col="0" width="5" font_point="11" bold="0" italics="0" align="left" type="number" >3    </data>
</line>
<line>
<data col="0" width="18" font_point="11" bold="1" italics="0" align="left" >Test Report (Bold)</data>
</line>
<line>
<data col="0" width="21" font_point="11" bold="0" italics="1" align="left" >Test Report (Italics)</data>
</line>
<line>
<data col="0" width="30" font_point="11" bold="1" italics="1" align="left" >Test Report (Bold and Italics)</data>
</line>
<line>
<data col="0" width="5" font_point="11" bold="1" italics="1" align="left" >BOO: </data>
<data col="0" width="10" font_point="11" bold="0" italics="0" align="left" type="string" >mike is a </data>
</line>
</output>
</part_header>
<part_page_header>
</part_page_header>
<table>
<tr>
<td width="100.000000" ><report_header>
</report_header>
<field_headers>
<output>
<line>
<data col="1" width="20" font_point="9" bold="0" italics="0" align="left" bgcolor="#e5e5e5" >First Name          </data>
<data col="0" width="1" font_point="9" bold="0" italics="0" align="left" bgcolor="#e5e5e5" > </data>
<data col="2" width="20" font_point="9" bold="0" italics="0" align="left" bgcolor="#e5e5e5" >Last Name           </data>
<data col="0" width="1" font_point="9" bold="0" italics="0" align="left" bgcolor="#e5e5e5" > </data>
<data col="3" width="20" font_point="9" bold="0" italics="0" align="left" bgcolor="#e5e5e5" >Color               </data>
</line>
</output>
</field_headers>
<break_header name="break0">
<output>
<line>
<data col="1" width="20" font_point="9" bold="0" italics="0" align="left" type="string" >1                   </data>
</line>
</output>
</break_header>
<field_details>
<output>
<line>
<data col="1" width="20" font_point="9" bold="0" italics="0" align="left" bgcolor="#ffff00" type="string" >Bob                 </data>
<data col="0" width="1" font_point="9" bold="0" italics="0" align="left" bgcolor="#ffffff" > </data>
<data col="2" width="20" font_point="9" bold="0" italics="0" align="left" bgcolor="#ffff00" type="string" >Doan                </data>
<data col="0" width="1" font_point="9" bold="0" italics="0" align="left" bgcolor="#ffffff" > </data>
<data col="3" width="20" font_point="9" bold="0" italics="0" align="left" bgcolor="#ffffff" type="string" >blue                </data>
<data col="0" width="1" font_point="9" bold="0" italics="0" align="left" bgcolor="#ffffff" > </data>
<data col="3" width="5" font_point="9" bold="0" italics="0" align="left" bgcolor="#ffffff" type="string" >WRD  </data>
<data col="0" width="1" font_point="9" bold="0" italics="0" align="left" bgcolor="#ffffff" > </data>
<data col="3" width="8" font_point="9" bold="0" italics="0" align="left" bgcolor="#ffffff" type="string" >MATCH   </data>
<data col="0" width="1" font_point="9" bold="0" italics="0" align="left" bgcolor="#ffffff" > </data>
<data col="3" width="8" font_point="9" bold="0" italics="0" align="left" bgcolor="#ffffff" type="string" >MATCH   </data>
<data col="0" width="1" font_point="9" bold="0" italics="0" align="left" bgcolor="#ffffff" > </data>
<data col="3" width="8" font_point="9" bold="0" italics="0" align="left" bgcolor="#ffffff" type="string" >MATCH   </data>
<data col="0" width="1" font_point="9" bold="0" italics="0" align="left" bgcolor="#ffffff" > </data>
<data col="3" width="8" font_point="9" bold="0" italics="0" align="left" bgcolor="#ffffff" type="string" >MATCH   </data>
</line>
</output>
</field_details>
<field_details>
<output>
<line>
<data col="1" width="20" font_point="9" bold="0" italics="0" align="left" bgcolor="#ffff00" type="string" >Terry               </data>
<data col="0" width="1" font_point="9" bold="0" italics="0" align="left" bgcolor="#e5e5e5" > </data>
<data col="2" width="20" font_point="9" bold="0" italics="0" align="left" bgcolor="#ffff00" type="string" >Doan                </data>
<data col="0" width="1" font_point="9" bold="0" italics="0" align="left" bgcolor="#e5e5e5" > </data>
<data col="3" width="20" font_point="9" bold="0" italics="0" align="left" bgcolor="#e5e5e5" type="string" >blue                </data>
<data col="0" width="1" font_point="9" bold="0" italics="0" align="left" bgcolor="#e5e5e5" > </data>
<data col="3" width="5" font_point="9" bold="0" italics="0" align="left" bgcolor="#e5e5e5" type="string" >TD   </data>
<data col="0" width="1" font_point="9" bold="0" italics="0" align="left" bgcolor="#e5e5e5" > </data>
<data col="3" width="8" font_point="9" bold="0" italics="0" align="left" bgcolor="#e5e5e5" type="string" >MATCH   </data>
<data col="0" width="1" font_point="9" bold="0" italics="0" align="left" bgcolor="#e5e5e5" > </data>
<data col="3" width="8" font_point="9" bold="0" italics="0" align="left" bgcolor="#e5e5e5" type="string" >MATCH   </data>
<data col="0" width="1" font_point="9" bold="0" italics="0" align="left" bgcolor="#e5e5e5" > </data>
<data col="3" width="8" font_point="9" bold="0" italics="0" align="left" bgcolor="#e5e5e5" type="string" >MATCH   </data>
<data col="0" width="1" font_point="9" bold="0" italics="0" align="left" bgcolor="#e5e5e5" > </data>
<data col="3" width="8" font_point="9" bold="0" italics="0" align="left" bgcolor="#e5e5e5" type="string" >MATCH   </data>
</line>
</output>
</field_details>
<field_details>
<output>
<line>
<data col="1" width="20" font_point="9" bold="0" italics="0" align="left" bgcolor="#ffff00" type="string" >Eric                </data>
<data col="0" width="1" font_point="9" bold="0" italics="0" align="left" bgcolor="#ffffff" > </data>
<data col="2" width="20" font_point="9" bold="0" italics="0" align="left" bgcolor="#ffff00" type="string" >Buruschkin          </data>
<data col="0" width="1" font_point="9" bold="0" italics="0" align="left" bgcolor="#ffffff" > </data>
<data col="3" width="20" font_point="9" bold="0" italics="0" align="left" bgcolor="#ffffff" type="string" >green               </data>
<data col="0" width="1" font_point="9" bold="0" italics="0" align="left" bgcolor="#ffffff" > </data>
<data col="3" width="5" font_point="9" bold="0" italics="0" align="left" bgcolor="#ffffff" type="string" >ERB  </data>
<data col="0" width="1" font_point="9" bold="0" italics="0" align="left" bgcolor="#ffffff" > </data>
<data col="3" width="8" font_point="9" bold="0" italics="0" align="left" bgcolor="#ffffff" type="string" >MATCH   </data>
<data col="0" width="1" font_point="9" bold="0" italics="0" align="left" bgcolor="#ffffff" > </data>
<data col="3" width="8" font_point="9" bold="0" italics="0" align="left" bgcolor="#ffffff" type="string" >MATCH   </data>
<data col="0" width="1" font_point="9" bold="0" italics="0" align="left" bgcolor="#ffffff" > </data>
<data col="3" width="8" font_point="9" bold="0" italics="0" align="left" bgcolor="#ffffff" type="string" >MATCH   </data>
<data col="0" width="1" font_point="9" bold="0" italics="0" align="left" bgcolor="#ffffff" > </data>
<data col="3" width="8" font_point="9" bold="0" italics="0" align="left" bgcolor="#ffffff" type="string" >MATCH   </data>
</line>
</output>
</field_details>
<break_footer name="break0">
<output>
<line>
<data col="0" width="6" font_point="9" bold="0" italics="0" align="left" >FOOTER</data>
<data col="1" width="20" font_point="9" bold="0" italics="0" align="left" type="string" >1                   </data>
</line>
</output>
</break_footer>
<break_header name="break0">
<output>
<line>
<data col="1" width="20" font_point="9" bold="0" italics="0" align="left" type="string" >2                   </data>
</line>
</output>
</break_header>
<field_details>
<output>
<line>
<data col="1" width="20" font_point="9" bold="0" italics="0" align="left" bgcolor="#ffff00" type="string" >Mike                </data>
<data col="0" width="1" font_point="9" bold="0" italics="0" align="left" bgcolor="#e5e5e5" > </data>
<data col="2" width="20" font_point="9" bold="0" italics="0" align="left" bgcolor="#ffff00" type="string" >Roth                </data>
<data col="0" width="1" font_point="9" bold="0" italics="0" align="left" bgcolor="#e5e5e5" > </data>
<data col="3" width="20" font_point="9" bold="0" italics="0" align="left" bgcolor="#e5e5e5" type="string" >yellow              </data>
<data col="0" width="1" font_point="9" bold="0" italics="0" align="left" bgcolor="#e5e5e5" > </data>
<data col="3" width="5" font_point="9" bold="0" italics="0" align="left" bgcolor="#e5e5e5" type="string" >     </data>
<data col="0" width="1" font_point="9" bold="0" italics="0" align="left" bgcolor="#e5e5e5" > </data>
<data col="3" width="8" font_point="9" bold="0" italics="0" align="left" bgcolor="#e5e5e5" type="string" >NO_MATCH</data>
<data col="0" width="1" font_point="9" bold="0" italics="0" align="left" bgcolor="#e5e5e5" > </data>
<data col="3" width="8" font_point="9" bold="0" italics="0" align="left" bgcolor="#e5e5e5" type="string" >NO_MATCH</data>
<data col="0" width="1" font_point="9" bold="0" italics="0" align="left" bgcolor="#e5e5e5" > </data>
<data col="3" width="8" font_point="9" bold="0" italics="0" align="left" bgcolor="#e5e5e5" type="string" >NO_MATCH</data>
<data col="0" width="1" font_point="9" bold="0" italics="0" align="left" bgcolor="#e5e5e5" > </data>
<data col="3" width="8" font_point="9" bold="0" italics="0" align="left" bgcolor="#e5e5e5" type="string" >NO_MATCH</data>
</line>
</output>
</field_details>
<field_details>
<output>
<line>
<data col="1" width="20" font_point="9" bold="0" italics="0" align="left" bgcolor="#ffff00" type="string" >Bob                 </data>
<data col="0" width="1" font_point="9" bold="0" italics="0" align="left" bgcolor="#ffffff" > </data>
<data col="2" width="20" font_point="9" bold="0" italics="0" align="left" bgcolor="#ffff00" type="string" >Kratz               </data>
<data col="0" width="1" font_point="9" bold="0" italics="0" align="left" bgcolor="#ffffff" > </data>
<data col="3" width="20" font_point="9" bold="0" italics="0" align="left" bgcolor="#ffffff" type="string" >pink                </data>
<data col="0" width="1" font_point="9" bold="0" italics="0" align="left" bgcolor="#ffffff" > </data>
<data col="3" width="5" font_point="9" bold="0" italics="0" align="left" bgcolor="#ffffff" type="string" >     </data>
<data col="0" width="1" font_point="9" bold="0" italics="0" align="left" bgcolor="#ffffff" > </data>
<data col="3" width="8" font_point="9" bold="0" italics="0" align="left" bgcolor="#ffffff" type="string" >NO_MATCH</data>
<data col="0" width="1" font_point="9" bold="0" italics="0" align="left" bgcolor="#ffffff" > </data>
<data col="3" width="8" font_point="9" bold="0" italics="0" align="left" bgcolor="#ffffff" type="string" >NO_MATCH</data>
<data col="0" width="1" font_point="9" bold="0" italics="0" align="left" bgcolor="#ffffff" > </data>
<data col="3" width="8" font_point="9" bold="0" italics="0" align="left" bgcolor="#ffffff" type="string" >NO_MATCH</data>
<data col="0" width="1" font_point="9" bold="0" italics="0" align="left" bgcolor="#ffffff" > </data>
<data col="3" width="8" font_point="9" bold="0" italics="0" align="left" bgcolor="#ffffff" type="string" >NO_MATCH</data>
</line>
</output>
</field_details>
<field_details>
<output>
<line>
<data col="1" width="20" font_point="9" bold="0" italics="0" align="left" bgcolor="#ffff00" type="string" >Steve               </data>
<data col="0" width="1" font_point="9" bold="0" italics="0" align="left" bgcolor="#e5e5e5" > </data>
<data col="2" width="20" font_point="9" bold="0" italics="0" align="left" bgcolor="#ffff00" type="string" >Tilden              </data>
<data col="0" width="1" font_point="9" bold="0" italics="0" align="left" bgcolor="#e5e5e5" > </data>
<data col="3" width="20" font_point="9" bold="0" italics="0" align="left" bgcolor="#e5e5e5" type="string" >purple              </data>
<data col="0" width="1" font_point="9" bold="0" italics="0" align="left" bgcolor="#e5e5e5" > </data>
<data col="3" width="5" font_point="9" bold="0" italics="0" align="left" bgcolor="#e5e5e5" type="string" >     </data>
<data col="0" width="1" font_point="9" bold="0" italics="0" align="left" bgcolor="#e5e5e5" > </data>
<data col="3" width="8" font_point="9" bold="0" italics="0" align="left" bgcolor="#e5e5e5" type="string" >NO_MATCH</data>
<data col="0" width="1" font_point="9" bold="0" italics="0" align="left" bgcolor="#e5e5e5" > </data>
<data col="3" width="8" font_point="9" bold="0" italics="0" align="left" bgcolor="#e5e5e5" type="string" >NO_MATCH</data>
<data col="0" width="1" font_point="9" bold="0" italics="0" align="left" bgcolor="#e5e5e5" > </data>
<data col="3" width="8" font_point="9" bold="0" italics="0" align="left" bgcolor="#e5e5e5" type="string" >NO_MATCH</data>
<data col="0" width="1" font_point="9" bold="0" italics="0" align="left" bgcolor="#e5e5e5" > </data>
<data col="3" width="8" font_point="9" bold="0" italics="0" align="left" bgcolor="#e5e5e5" type="string" >NO_MATCH</data>
</line>
</output>
</field_details>
<break_footer name="break0">
<output>
<line>
<data col="0" width="6" font_point="9" bold="0" italics="0" align="left" >FOOTER</data>
<data col="1" width="20" font_point="9" bold="0" italics="0" align="left" type="string" >2                   </data>
</line>
</output>
</break_footer>
</td>
</tr>
</table>
</part>
</rlib>
